zipkin-local.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+const _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+const {
+  THEME_OPTIONS,
+} = require("gatsby-plugin-newrelic-test/utils/constants");
+const { getCiData } = require("./utils/getCiData");
+const ciAttributes = getCiData();
+exports.__esModule = true;
+exports.stop = exports.create = void 0;
+
+const _zipkin = _interopRequireDefault(require("zipkin"));
+
+const _zipkinTransportHttp = require("zipkin-transport-newrelic");
+
+const _zipkinJavascriptOpentracing = _interopRequireDefault(
+  require("zipkin-javascript-opentracing")
+);
+const _nodeFetch = _interopRequireDefault(require("node-fetch"));
+let logger;
+let recorder;
+/**
+ * Create and return an open-tracing compatible tracer. See
+ * https://github.com/opentracing/opentracing-javascript/blob/master/src/tracer.ts
+ */
+
+const create = () => {
+  logger = new _zipkinTransportHttp.HttpLogger({
+    // endpoint of local docker zipkin instance
+    endpoint: `https://${
+      THEME_OPTIONS.staging ? `staging-` : ``
+    }trace-api.newrelic.com/trace/v1`,
+    headers: {
+      "Api-Key": THEME_OPTIONS.NR_INGEST_KEY,
+      "Data-Format": "zipkin",
+      "Data-Format-Version": 2,
+      options: THEME_OPTIONS,
+      tags: {
+        ...THEME_OPTIONS.traces.tags,
+        ...ciAttributes,
+      },
+    },
+  });
+  recorder = new _zipkin.BatchRecorder({
```

Comment:
To promote function purity (see above).

```suggestion
  const recorder = new _zipkin.BatchRecorder({
```